tests/vmi_monitoring_test.go

```diff
@@ -48,10 +47,8 @@ var _ = Describe("[sig-compute]Health Monitoring", func() {
 
 	Describe("A VirtualMachineInstance with a watchdog device", func() {
 		It("[test_id:4641]should be shut down when the watchdog expires", func() {
-			vmi := tests.NewRandomVMIWithWatchdog()
-			obj, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
-			Expect(err).To(BeNil())
-			tests.WaitForSuccessfulVMIStart(obj)
+			vmi := libvmi.NewAlpine(withWatchdog())
+			vmi = tests.RunVMIAndExpectLaunch(vmi, 360)
```

Comment:
Any reason for this change?

Reply:
I dislike the boilerplate of
```
			obj, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
			Expect(err).To(BeNil())
			tests.WaitForSuccessfulVMIStart(obj)
```
let alone the meaningless variable name `obj`.

Do you mean that I should have mentioned in the commit message that this change is intentional?

Reply:
All good. My brain ignored the `Create` call and thought this is just exposing timeout.